src/app/pages/storage/volumes/volume-status/volume-status.component.ts

```diff
@@ -508,7 +508,7 @@ export class VolumeStatusComponent implements OnInit, OnDestroy {
               entityDialog.dialogRef.close(true);
               this.getData();
               this.getUnusedDisk();
-              this.dialogService.info(helptext.extend_disk.title, helptext.extend_disk.info_dialog_content + name + '.', '', 'info', true);
+              this.dialogService.info(helptext.extend_disk.title, helptext.extend_disk.info_dialog_content + name + '.', '', 'info', true); // eslint-disable-line no-restricted-globals
```

Comment:
Clearly a mistake and original intend was to put a name of vdev into the message.